### PR TITLE
added kaviraj.eth profile

### PR DESCRIPTION
**Pull request by kaviraj-j**
## Description

This PR adds a new builder profile for kaviraj.eth (0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd)

## Related Issues

Issue #8 

Your ENS/address: kaviraj.eth

### CODE REVIEW

File: packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+
+export default function KavirajProfilePage() {
```
Comment (carletex) [inline packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx:3]: Could you use `const` here and export it at the bottom of the file? Just to be consistent within the project.

File: packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const bio =
+    "Hi my name is Kaviraj (kaviraj.eth) I'm a computer science student who builds web applications and interested in web3 space :)";
+  const links = [
+    { label: "Portfolio", url: "https://kaviraj-j.me" },
+    { label: "GitHub", url: "https://github.com/kaviraj-j" },
+    { label: "Twitter", url: "https://twitter.com/kaviraj_jk" },
+  ];
+
+  return (
+    <div className="container mx-auto px-4">
```
Comment (carletex) [inline packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx:14]: Maybe add some top padding here? If not it's touching the header

image

**Review by carletex — CHANGES_REQUESTED** [review]
Looking great @kaviraj-j !

Just left some tiny review comments

File: packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx (hunk 2 above)
Comment (kaviraj-j) [reply packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx:14]: Yeah, I'll do it and give a PR again!

Comment (kaviraj-j) [reply packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx]: Hey man, I just committed the repo with those changes, do I have to give a new PR?

Comment (carletex) [reply packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx]: No, this is perfect! The PR gets updated automatically ;)

**Review by carletex — APPROVED** [review]

**Pull request merged**